scanner/cmd/scannerctl/sbom.go

```diff
@@ -64,8 +65,9 @@ func sbomCmd(ctx context.Context) *cobra.Command {
 				ref.DigestStr(), *imageDigest)
 		}
 
+		uri := fmt.Sprintf("%s-%s", imageURL, uuid.NewV4().String())
```

Comment:
honestly not sure if this is best vs `imageURL + "-" + uuid.NewV4().String()`

Reply:
https://github.com/stackrox/stackrox/pull/14135/commits/051516a2d713d27efb3a14453d28f30ce618a317